### DISCUSSION TITLE
pool idea.

### DISCUSSION THREAD

**Discussion by claudio782** [Chia-Network/chia-blockchain#4382] [Ideas]
create a unique pool code.
When creating the plot, associate it with the unique pool code
if the plot of a certain pool is the winner, divide the prize in all the plots of that pool and
pay the prize in proportion to the amount of plot that each farmer contributes

the more plots they bring to the pool, the more profits the farmers will get

Another benefit is that you could have plots in several pools at the same time

**Comment by SackofFluor** [comment]
There are already 2 unofficial pools and according to a comment or two people are not really happy with it.
The dev-team has recently announced that there will be an official pool protocol kinda soon. Even with GUI support. So lets see how this is turning out.

**Reply by claudio782** [reply]
It would be good if they manage to find the balance so that both small and large farmers benefit.

**Comment by gushome7** [comment]
That is a problem, not a benefit. If everybody can have same plots in 
several  pools (why not in all?) every block reward will be divided 
between all farmers, and that will destroy farming and chia network.

**Reply by claudio782** [reply]
You may not express the concept well. The idea is not to have the same plot in several pools. what I meant is that in the same storage unit you could have plots for different pools.

**Comment by brezapour81** [comment]
If people are interest, we are launching a Chia Pool soon called https://Seedpool.org. Check it out.

**Reply by SackofFluor** [reply]
Does it use the to be announced pool protocol?
OR is it just a new grift pool?

**Reply by brezapour81** [reply]
We will be using official Chia pooling protocol.

**Reply by salam836** [reply]
@brezapour81 how do i join ?

**Reply by brezapour81** [reply]
Hi salam836, 

Please visit www.seedpool.org and subscribe for details of launch date.
